nautobot/docs/user-guide/platform-functionality/jobs/managing-jobs.md

```diff
@@ -38,7 +38,6 @@ Sometimes you need to change how a Job appears or behaves without touching its s
 | **grouping** | UI category the Job is listed under. | Organize third-party Jobs into "Compliance" or "Local Automations." |
 | **name** | Display name in the Jobs list. | Rename technical Job names to user-friendly descriptions. |
 | **description** | Markdown or HTML description shown on Job detail page. | Add instructions or documentation links. |
-| **approval_required** | Require an additional user approval before each execution. | Protect Jobs that perform bulk production changes. |
```

Comment:
Do we have any content/cross-references in this document about approval workflows? If not we need to add that at some point.